### ISSUE TITLE
Link for transfering beers too long

### ISSUE THREAD

**Issue by petterl** [jonpacker/cbc-menu#1]
After attending a MBCC the base64 generated in the link is too long to be able to be pasted into another browser.

Maybe a packing algorithm can be used or other way of sending the info.

The workaround I had to use was to go into the console and run the localstore commands manually to transfer the info from my phone to computer.

**Comment by petterl** [comment]
After MBCC my url var 29374 characters long. No browser accepts it.

Should we try to compress the data using compression or do you have any other solution-idea?

**Comment by jonpacker** [comment]
Sorry it took me so long to get around to responding to these. My solution to this would be to remove it altogether, and rely solely on untappd snapshots. Maybe make it possible to save and load a snapshot somehow without connecting to untappd.

**Comment by petterl** [comment]
My issue whas that I never managed to connect my Iphone with crome to untappd, it was just spinning (no java console to look at =( ) So in the end a transfered it all to a computer and connected and it worked fine.

**Comment by jonpacker** [comment]
Yeah, I see. Sorry about that. That URL transfer method is definitely not ideal.
